common/djangoapps/student/management/commands/export_staff_users.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import absolute_import, print_function
+
+import logging
+from datetime import datetime, timedelta
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.core.mail.message import EmailMultiAlternatives
+from django.template.loader import get_template
+from os import remove
+
+from djqscsv import write_csv
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from student.models import CourseAccessRole
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+
+    help = """
+    This command will export a csv of all users who have logged in within the given days and
```

Comment:
Please add an example of how we are going to use the command. 
https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades/management/commands/recalculate_learner_grades.py#L17